clang/test/SemaHLSL/BuiltIns/any-errors.hlsl

```diff
@@ -1,5 +1,5 @@
 
-// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -verify -verify-ignore-unexpected
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -verify -verify-ignore-unexpected -o /dev/null
```

Comment:
Unless I'm mistaken the clang `-verify` option should prevent writing outputs since it is just running the verifier.

That said, this test shouldn't be passing `-emit-llvm -disable-llvm-passes` because we aren't actually checking the IR.

Reply:
This problem is repeated in a bunch of the other updated tests. I'm not going to comment on each one individually.

Reply:
I think it is `-emit-llvm` that force output to be written even with `-verify`. The correct thing, as you pointed out, is not to add `-o /dev/null` but to remove `-emit-llvm -disable-llvm-passes`. I will updated the pull request accordingly.

Reply:
Removing `-emit-llvm` will expose more of the compiler pipeline to the testcases and some testcases seems not to be designed to work with that. Instead I found out that there is another option `-emit-llvm-only` that seems to be designed to be used together with `-verify` as it to the same thing as `-emit-llvm` except that it don't produce any output.

Is that an acceptable solution?